src/pages/all-about-carbon-for-ibm.com/what-is-carbon-for-ibm.com.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: What is Carbon for IBM.com?
+description:
+  Carbon for IBM.com is the design system for IBM.com's digital
+  experiences. The system consists of working code, design tools, resources and
+  it's tailored to IBM.com website page makers.
+---
+
+<PageDescription>
+
+Carbon for IBM.com is the open source design system for IBM.com's digital experiences. The system consists of working code, design tools, and resources and is tailored to IBM.com website page makers.
+
+</PageDescription>
+
+## What is Carbon for IBM.com?
+
+As IBM.com's official design system, Carbon for IBM.com serves a wide range of designers and
+developers that are building digital products and experiences.
+
+Carbon for IBM.com is building for the IBM.com maker's business needs. It also means we have a brilliant team of engineers
+and designers working full time to help make IBM.com better. It is maintained by the Digital Design Squad, a
+core team of designers, developers, and writers. 
+
+The goals of the Carbon for IBM.com include improving visual consistency and quality for all IBM.com website pages,
+establishing a shared user-centered experience for all IBM.com visitors, and providing
+clear, discoverable guidance around design and development best practices to all Carbon for IBM.com adopters.
```

Comment:
Should this just be "IBM.com website page makers" ?

Reply:
@shixiedesign Which particular part are you referring to?

Reply:
Line 38?